docs/reference/verify-assets.mdx

```diff
@@ -305,7 +302,181 @@ $ sudo ig daemon --verify-image=false
+            MA4GA1UdDwEB/wQEAwIHgDATBgNVHSUEDDAKBggrBgEFBQcDAzANBgkqhkiG9w0B
+            AQsFAAOCAQEANjfgsg6Zxc/g+8T3ZLNMj0myd8yciiL16iS4+LQKViRGNmAjRbic
+            eslrAFXSiZHI/RIJxKnQBxtDiXs8UlknV2mLnxZP+S2vk3kW/z0w6gRPRC8t3w3K
+            BcsDhE//PzLmnXdEYSZQvgXfdv2JujEb1euspyTjRL1vBlxdPh5eeevZsysHrt6q
+            c2bFCX/8nicguibCC4ybKQazsbQK07tIPDwlbu0+sCk6ojmhdMZeOaXVcUoaisgq
+            dKl8IemhdshLNii2f05fLb1ftcMaG0XIT7i86txiDbiBs+srcjW4bqfhSc201jHu
+            /ftuKYyrrWBgnWZ6gmzqC5SR4pBdZVAp3w==
+            -----END CERTIFICATE-----
+        notation-policy-document: |
+          {
+              "version": "1.0",
+              "trustPolicies": [
+                  {
+                      "name": "inspektor-gadget-policy",
+                      "registryScopes": [ "*" ],
+                      "signatureVerification": {
+                        "level" : "strict"
+                      },
+                      "trustStores": ["ca:foobar.io"],
+                      "trustedIdentities": [
+                        "x509.subject: CN=foobar.io, O=Notary, L=Seattle, ST=WA, C=US"
+                      ]
+                  }
+              ]
+          }
+EOF
+```
+
+Then, you can deploy Inspektor Gadget with this configuration:
+
+```bash
+
+$ kubectl gadget deploy --daemon-config=daemon-config.yaml
+...
+Inspektor Gadget successfully deployed
+$ kubectl-gadget run ghcr.io/your_repo/trace_exec -A
+K8S.NAMESP… K8S.PODNAME K8S.CONTAI… MNTNS… TIMES… PID    PPID   UID    GID    LOGIN… SESSI… RETVAL ARGS_… UPPER… ARGS_… COMM  ARGS  K8S.…
+gadget      gadge…hbh8n gadget      40265… 22867… 53386  53368  0      0      42949… 42949… 0      2      false  35     gadg… /bin… mini…
+gadget      gadge…hbh8n gadget      40265… 22867… 53387  53369  0      0      42949… 42949… 0      2      false  35     gadg… /bin… mini…
+...
+```
+
+You can also instruct Inspektor Gadget to skip the verification by disabling it at deploy time.
+Please note that we do not recommend doing so and that it should only be used for development purposes.
+Again start by creating a daemon config file:
+
+```bash
+cat <<EOF > daemon-config.yaml
+operator:
+    oci:
+        verify-image: false
+EOF
+```
+
+Then, you can deploy Inspektor Gadget with this configuration:
+
+```bash
+$ kubectl gadget deploy --daemon-config=daemon-config.yaml
+...
+Inspektor Gadget successfully deployed
+$ kubectl gadget run trace_exec -A
+WARN[0001] minikube-docker      | gadget signature verification is disabled due to using corresponding option
+K8S.NAMESPACE       K8S.PODNAME         K8S.CONTAINERNAME          PID       PPID RE… COMM      ARGS      K8S.NODE  TIMESTAMP
+gadget              gadget-z55jq        gadget                   55376      55357   0 gadgettr… /bin/gad… minikube… 2024-07-17T07:39:07.…
+gadget              gadget-z55jq        gadget                   55375      55358   0 gadgettr… /bin/gad… minikube… 2024-07-17T07:39:07.…
+...
+```
+</TabItem>
```

Comment:
This is duplicated in both notation and cosign. I think it should be a third "disable verification" section instead.